proxy/src/control/pb.rs

```diff
@@ -78,8 +77,7 @@ impl event::StreamResponseFail {
         };
 
         let destination_meta = ctx.dst_labels()
-            .and_then(|b| b.borrow().clone())
-            .map(|d| tap_event::EndpointMeta {
+            .map(|ref d| tap_event::EndpointMeta {
```

Comment:
It would be nice if we could, in a subsequent PR, factor out all the repetition evident from the changes to this file, since this appears to be the same code fragment over and over again.